test/e2e/util.go

```diff
@@ -333,7 +341,7 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	kubectlArgs := append(defaultArgs, args...)
 
 	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command("kubectl", kubectlArgs...)
+	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
```

Comment:
Actually, we have been testing gke with the version of kubectl from the release. I think backwards compatibility is now the thing we need to test more than future breakages, so we should probably use the bundled kubectl.